Documentation/Configuration.md

```diff
@@ -0,0 +1,23 @@
+# Configuring WheelMUD
+
+## Server Data
+When building and running the server for the first time, a bunch of files are copied over to the server's "program data" location where it can persist and evolve (such as when new characters are saved to the database).
```

Comment:
Maybe this is a bug? Do we want people to be able to more easily change/backup/store their data?